### PR TITLE
Update Readme for 2.0

### PR DESCRIPTION
**Pull request by s-martin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +124,20 @@ The **web app** runs on any device and is mobile optimised. It provides:
 
 Adding push buttons to control volume, skipping tracks, pause, play: read the [GPIO buttons installation guide](docs/GPIO-BUTTONS.md).
```
Comment (MiczFlor) [inline README.md:125]: Would it be better to move the GPIO doc to the wiki and link to the wiki page?

File: README.md [hunk 2]
```diff
@@ -114,6 +124,20 @@ The **web app** runs on any device and is mobile optimised. It provides:
...
 
+### Components
```
Comment (MiczFlor) [inline README.md:127]: Not sure we should have this in the readme file. It feels to detailed. Possibly just one line and then actually linking to the folder `components` in the code. Note: link would need to point to master not `develop` once we released 2.0

File: README.md [hunk 3]
```diff
@@ -23,17 +23,27 @@ In case you missed it, [download the 2019 Phoniebox Calendar PDF here](https://d
 
 * **Gitter Community** we got ourselves a gitter community; chat us up at https://gitter.im/phoniebox
 
-* **Phoniebox 2.0 rc8 released (2020-01-18 ... getting closer to the actual release...)
+* **Phoniebox [2.0](https://github.com/MiczFlor/RPi-Jukebox-RFID/milestone/1?closed=1) released (2020-05-xx)**
```
Comment (MiczFlor) [inline README.md:27]: I would like to add a line here:
The 2.0 release was pushed through the doors with *countless additions and great oversight* from @s-martin. Thank you. And *substantial input* by @ZyanKLee and [many more contributors](https://github.com/MiczFlor/RPi-Jukebox-RFID/pulls?q=is%3Apr) like, in alphabetical particular order: @andreasbrett @BerniPi  @danielhammer @drocx @fredg02 @Groovylein @lovethisshit @MalteHST @Piehti @SimonChelkowski @veloxidSchweiz and starring @splitti as *the great hotfix*.

File: README.md (hunk 2 above)
Comment (s-martin) [reply README.md:127]: Ok for me.
Links are only relative, so in master they would point to the correct master file.

File: README.md (hunk 1 above)
Comment (s-martin) [reply README.md:125]: Yes, we can do that.
But I think the wiki needs some editing, too. E.g. many of manual steps described aren't necessary anymore with the install script.

**Comment by s-martin** [comment]
@miczflor, just update more as you like.

**Pull request merged**